Dockerfile

```diff
@@ -3,4 +3,5 @@ FROM rancher/dind:v0.2.0
 COPY ./scripts/bootstrap /scripts/bootstrap
 RUN /scripts/bootstrap
 WORKDIR /source
+VOLUME /target
```

Comment:
@sonchang Valid point but that wouldn't matter here as this is a volume for the container only this would matter if when the container is run /target is bind mounted to the host. Which is what  build master would do. So as a result the most recent run would always be the file that is used if this were bind mounted.